bogota-abril-2019/css-magazine-cover/federico prieto/index.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+  <meta charset="utf-8">
+  <title></title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="manifest" href="site.webmanifest">
+  <link rel="apple-touch-icon" href="icon.png">
+  <!-- Place favicon.ico in the root directory -->
+
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Fjalla+One|Old+Standard+TT" rel="stylesheet">
+  <meta name="theme-color" content="#fafafa">
+</head>
+
+<body>
+  <!--[if IE]>
+    <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
+  <![endif]-->
+
+  <!-- Add your site or application content here -->
+  <div class="outerContainer">
+    <div class="innerContainer">
+      <header>
+        <h1><span>NATIONAL</span>GEOGRAPHIC</h1>
+      </header>
+      <section>
+        <div class="leftTitles">
+          <ul>
+            <li>
+              <h2>GREAT SALT LAKE: <br> THE FLOODING <br> DESERT <span>694</span></h2>
```

Comment:
Unnecessary used of Capital letter. Please use Text-transform CSS property.

https://css-tricks.com/almanac/properties/t/text-transform/